cabot/settings.py

```diff
@@ -227,6 +230,10 @@
     }
 }
 
+# Disable logging for tests
```

Comment:
I could disable logging in tests_basic.py also, but Jim recommended against it because it adds an import-time side effect to tests. Not sure where else I could do this